server/storage/key_path.go

```diff
@@ -8,29 +8,31 @@ import (
 )
 
 const (
-	cluster       = "v1/cluster"
+	version       = "v1"
+	cluster       = "cluster"
 	schema        = "schema"
 	table         = "table"
 	shard         = "shard"
+	node          = "node"
 	topology      = "topo"
 	latestVersion = "latest_version"
 )
 
 // makeSchemaKey returns the key path to the schema meta info.
 // example:
-// cluster 1: v1/cluster/1/schema/1 -> pb.Schema
-//            v1/cluster/1/schema/2 -> pb.Schema
-//            v1/cluster/1/schema/3 -> pb.Schema
+// cluster 1: v1/cluster/1/schema/schema/1 -> pb.Schema
```

Comment:
It is weird to double the schema, and maybe we should call the second `schema` as `info`. And this is the same for doubled `cluster`.